src/autocomplete/databases/redis/grammar/RedisParser.g4

```diff
@@ -0,0 +1,59 @@
+// $antlr-format columnLimit 500, minEmptyLines 1, maxEmptyLinesToKeep 1, useTab false, reflowComments false, breakBeforeBraces false
+// $antlr-format keepEmptyLinesAtTheStartOfBlocks false, allowShortRulesOnASingleLine false, alignSemicolons hanging, alignColons hanging
+// $antlr-format alignTrailingComments true
+
+parser grammar RedisParser;
+
+options {
+    tokenVocab = RedisLexer;
+}
+
+root
+    : commands? EOF
+    ;
+
+// We can omit newline only if it's the last statement
+commands
+    : command NEWLINE?
+    | command NEWLINE commands
+    ;
+
+command
```

Comment:
Are you sure we should call it `command`, and not `statement`? `Statements` is a common name afaik

Reply:
that's the way the call it everywhere in regards to Redis (including official docs), so I thought we better stick to conventional naming
<img width="1076" alt="Screenshot 2024-08-07 at 17 06 51" src="https://github.com/user-attachments/assets/007ab2f7-894a-4276-ada0-5605c5da35b9">

Reply:
Nice one!